### PR TITLE
Extend yast_hostnames for adding/changing hostname

### PR DESCRIPTION
**Pull request by Zaoliang**
Please see reference tests:
openSUSE TW
http://e13.suse.de/tests/2968#step/yast2_hostnames

SLES 12 SP3
http://e13.suse.de/tests/2965#step/yast2_hostnames

### CODE REVIEW

File: tests/yast2_gui/yast2_hostnames.pm [hunk 1]
```diff
@@ -22,9 +22,25 @@ sub run() {
     my $self   = shift;
     my $module = "host";
 
+    select_console 'root-console';
+    #	add 1 entry to /etc/hosts and edit it later
+    script_run "echo '10.160.1.100    dist.suse.de dist' >> /etc/hosts";
```
Comment (okurz) [inline tests/yast2_gui/yast2_hostnames.pm]: I don't think it's a good idea to share the SUSE internal IP address here.

Comment (Zaoliang) [inline tests/yast2_gui/yast2_hostnames.pm]: okay, will change this.

**Comment by Zaoliang** [comment]
@okurz please check and merge, thanks!

**Comment by Zaoliang** [comment]
tftp-server needles doesn't match anymore, 82%
http://e13.suse.de/tests/2974#step/yast2_control_center/10

File: tests/yast2_gui/yast2_hostnames.pm [hunk 2]
```diff
@@ -22,9 +22,25 @@ sub run() {
     my $self   = shift;
     my $module = "host";
 
+    select_console 'root-console';
+    #	add 1 entry to /etc/hosts and edit it later
+    script_run "echo '80.92.65.530    n-tv.de ntv' >> /etc/hosts";
```
Comment (dzedro) [inline tests/yast2_gui/yast2_hostnames.pm]: invalid IP ?

Comment (Zaoliang) [reply tests/yast2_gui/yast2_hostnames.pm]: should be 80.92.65.53

Comment (Zaoliang) [reply tests/yast2_gui/yast2_hostnames.pm]: PR: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5424

Thanks!

**Pull request merged**